recipe/conda_build_config.yaml

```diff
@@ -478,12 +480,24 @@ libboost_headers:
   - '1.84'
 libboost_python_devel:
   - '1.84'
+libbtf:
+  - '2'
+libcamd:
+  - '3'
 libcint:
   - '6.1'
+libccolamd:
+  - '3'
+libcholmod:
+  - '5'
+libcolamd:
+  - '3'
```

Comment:
I'm of two minds if we don't want to namespace these as coming from suitesparse somehow. I know that suitesparse more or less independent projects (hence "suite"), but we're not going to get these from anywhere else realistically? No strong opinions, just wondering...

Reply:
I wondered that, too! Lacking other reasons, I went with using the same names as ubuntu (`suitesparse-` is on mongoose, but not the others, I think because there are other pre-existing projects with the same name).